pkg/rewards/10_goldAvsODRewardAmounts.go

```diff
@@ -0,0 +1,78 @@
+package rewards
+
+import (
+	"github.com/Layr-Labs/sidecar/pkg/rewardsUtils"
+	"go.uber.org/zap"
+)
+
+const _10_goldAvsODRewardAmountsQuery = `
+create table {{.destTableName}} as
+-- Get the rows where operators have not registered for the AVS or if the AVS does not exist
+WITH reward_snapshot_operators as (
+  SELECT
+    ap.reward_hash,
+    ap.snapshot as snapshot,
+    ap.token,
+    ap.tokens_per_day,
+    ap.tokens_per_day_decimal,
+    ap.avs as avs,
+    ap.operator as operator,
+    ap.strategy,
+    ap.multiplier,
+    ap.reward_submission_date
+  FROM {{.activeRewardsTable}} ap
+  LEFT JOIN operator_avs_registration_snapshots oar
+  ON ap.avs = oar.avs and ap.snapshot = oar.snapshot and ap.operator = oar.operator
```

Comment:
style nit:

If the `on` is going to go on a new line, it should be indented. Alternatively, keep it on the same line as the join and put the conditions on a newline each. Also the condition should be enclosed in parenthesis. Makes it easier to read.

```sql
LEFT JOIN operator_avs_registration_snapshots oar ON (
    ap.avs = oar.avs
    and ap.snapshot = oar.snapshot
    and ap.operator = oar.operator
)
```

Reply:
I just followed the convention on the existing queries. But happy to modify this.

Reply:
I can lint it separately and then copy it over.

Reply:
Done in [b3f5ebe](https://github.com/Layr-Labs/sidecar/pull/131/commits/b3f5ebe465a23b609b3b3e804a45148361c7af47)